src/components/UI/Form/TimePicker/TimePicker.tsx

```diff
@@ -43,7 +43,6 @@ export const TimePicker: React.SFC<TimePickerProps> = ({
       <Grid className={styles.TimePicker}>
         <KeyboardTimePicker
           autoOk
-          error={hasError ? errorText : ''}
```

Comment:
Same here

Reply:
Reverted this too. I think it works as it passes all props to the underlying text component.